src/psyclone/domain/lfric/lfric_builtins.py

```diff
@@ -310,10 +311,20 @@ def reference_element(self):
         '''
         return None
 
-    @abc.abstractmethod
     def gen_code(self, parent):
-        ''' Must be implemented in sub-class. Will generate the f2pygen AST
-        for this builtin. '''
+        '''
+        Generates LFRic API specific PSy code (in the form of f2pygen AST
+        nodes) for a call to this Built-in. This method must be overridden
+        if a sub-class does not yet implement lower_to_language_level().
+
+        :param parent: Node in f2pygen tree to which to add call.
+        :type parent: :py:class:`psyclone.f2pygen.BaseGen`
+
+        '''
+        # Ultimately this routine will be removed once the LFRic PSyIR
+        # has been fully migrated to use the PSyIR backends (e.g. #1010).
```

Comment:
Bring the issue number at the front and mark this as a TODO.

Reply:
Done.